internal/pkg/model/ext.go

```diff
@@ -25,3 +25,16 @@ func (m *Server) Time() (time.Time, error) {
 func (m *Server) SetTime(t time.Time) {
 	m.Timestamp = t.Format(time.RFC3339Nano)
 }
+
+// GetNewVersion returns Agent version if it is different from ver, otherwise return empty string
+func (m *Agent) GetNewVersion(ver string) string {
```

Comment:
Nit: It works also for older versions and only new :-)

Reply:
> Nit: It works also for older versions and only new :-)
yeah might be the best name for the function, open to suggestions.